rosdep/base.yaml

```diff
@@ -1350,6 +1355,11 @@ libexpat1-dev:
     portage:
       packages: [dev-libs/expat]
   ubuntu: [libexpat1-dev]
+libfcl-dev:
+  arch: [fcl]
+  debian: [libfcl-dev]
+  fedora: [fcl]
```

Comment:
If we want to skip fedora support, should I remove this line or just give empty brackets?

~~~
fedora: []
~~~